### DISCUSSION TITLE
DLNA on Docker Desktop for Windows with WSL2

### DISCUSSION THREAD

**Discussion by sanctimon** [jellyfin/jellyfin#10192] [General]
The official documentation suggests that for DLNA to work on Docker, the network selected must be "host". However, the host driver is not functional in Docker Desktop for Windows.

Is there another way to enable DLNA when running jellyfin on docker desktop for Windows?

**Comment by Shadowghost** [comment]
DLNA/SSDP relies on multicast/broadcast which is not usable if using port forwarding (because NAT is involved).

Not sure if Docker on Windows supports things like macvlan for networking which can make this functioning without using host networking too.
